src/types/polygon.rs

```diff
@@ -70,6 +72,58 @@ where
     }
 }
 
+impl<T: WktNum> PolygonTrait for Polygon<T> {
+    type T = T;
+    type RingType<'a> = &'a LineString<T> where Self: 'a;
+
+    fn dim(&self) -> geo_traits::Dimensions {
+        // TODO: infer dimension from empty WKT
```

Comment:
Can you please make an issue describing the work to be done here and any context that we might not remember in a hundred years when someone actually does the work?

Reply:
Made https://github.com/georust/wkt/issues/125